docs/gallery/cnot.ipynb

```diff
@@ -1367,32 +203,25 @@
     "        ax.loglog()\n",
     "        ax.set_title(\"Logical CNOT Error Rate\")\n",
     "        ax.set_xlabel(\"Physical Error Rate\")\n",
-    "        ax.set_ylabel(\"Logical Error Rate\")\n",
-    "        fig.savefig(\n",
-    "            SAVE_DIR\n",
-    "            / f\"logical_cnot_result_{support_observable_basis}_observable_{i}.png\"\n",
-    "        )\n",
-    "\n",
-    "\n",
-    "def main():\n",
-    "    SAVE_DIR.mkdir(exist_ok=True)\n",
-    "    generate_graphs(Basis.Z)\n",
-    "    generate_graphs(Basis.X)\n",
-    "\n",
-    "\n",
-    "if __name__ == \"__main__\":\n",
-    "    main()\n",
-    "```\n",
-    "\n",
-    "</details>\n",
-    "\n",
-    "![ZIZI](../media/gallery/cnot/IXIX.png)\n",
-    "\n",
-    "![ZZIZ](../media/gallery/cnot/XIXX.png)\n",
-    "\n",
-    "![ZIZI](../media/gallery/cnot/ZIZI.png)\n",
-    "\n",
-    "![ZZIZ](../media/gallery/cnot/ZZIZ.png)"
+    "        ax.set_ylabel(\"Logical Error Rate\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "generate_graphs(Basis.Z)"
```

Comment:
To do: Hide the pymatching progress output

https://tqec.github.io/tqec/pull/592/gallery/cnot.html#Example-Circuit